interface/resources/qml/hifi/dialogs/security/EntityScriptQMLWhitelist.qml

```diff
@@ -129,30 +167,39 @@ Rectangle {
           id: notificationText;
           text: ""
           // Text size
-          size: 14;
+          size: 16;
           // Style
           color: "white";
           elide: Text.ElideLeft;
           // Anchors
-          anchors.right: parent.right;
-          anchors.rightMargin: 130;
+          anchors.right: parent.left;
+          anchors.rightMargin: 10;
       }
     }
     
     HifiStylesUit.RalewayRegular {
         id: descriptionText;
-        text: "Separate your URLs by line, not commas. Example:
-        https://google.com/
-        https://bing.com/
-        https://mydomain.here/
-        \nEnsure there are no spaces or whitespace.
-        \nFor QML files, you can only whitelist each file individually 
-        ending with '.qml'."
+        text: 
+"The whitelist checks scripts/QML as it is loaded.<br/>
+Therefore, if a script is cached or has no reason to load again,<br/>
+then removing it from the whitelist will not be effective until<br/>
+it is reloaded.<br/>
+Separate your whitelisted domains by line, not commas. e.g.
```

Comment:
Separate your whitelisted domains by line, not commas. Example:

Reply:
Would love to but the tablet is only so wide, so my text is usually weird to accommodate that phenomenon.